framework/tests/packets.rs

```diff
@@ -129,291 +123,296 @@ fn packet_from_bytes(bytes: &[u8]) -> Packet<NullHeader, EmptyMetadata> {
     pkt
 }
 
+#[test]
 fn srh_from_bytes() {
-    let pkt = packet_from_bytes(&SRH_BYTES);
-
-    // Check Ethernet header
-    let epkt = pkt.parse_header::<MacHeader>();
-    {
-        let eth = epkt.get_header();
-        assert_eq!(eth.dst.addr, MacAddress::new(0, 0, 0, 0, 0, 1).addr);
-        assert_eq!(eth.src.addr, MacAddress::new(0, 0, 0, 0, 0, 2).addr);
-        assert_eq!(eth.etype(), Some(EtherType::IPv6));
-    }
+    dpdk_test! {{
```

Comment:
A little ugly. I don't know how to get rid of one of the `{}`.

Reply:
Let's vote, which is better?

1. `{{ }}`
2. `({ })`
3. `[{ }]`

Reply:
2

Reply:
and yes, don't think you can get around block without `{}` from what i've seen typically.

Reply:
Ok, don't need to vote anymore. Figured it out. No more doubling up.